community/mbedtls/mbedtls-utils.profile.d

```diff
@@ -0,0 +1 @@
+export PATH=$PATH:/usr/libexec/mbedtls
```

Comment:
Why not install mbedtls in `/usr/bin` if it is supposed to be in `PATH`?

Why is it installed in `/usr/libexec` in the first place?

Reply:
I thought you put them in `libexec` for a reason - have moved the `utils` into `/usr/bin`